docs/self-managed/zeebe-deployment/operations/backups.md

```diff
@@ -5,6 +5,10 @@ description: "A guide to creating and installing Zeebe backups."
 keywords: ["backpressure", "back-pressure", "back pressure"]
```

Comment:
Not your fault (I think mine actually :sweat_smile:) but these keywords should be adjusted.

Reply:
:thinking: Strange! I had updated this.